plugins/org.eclipse.xtext.util/src/org/eclipse/xtext/util/Strings.java

```diff
@@ -598,4 +599,20 @@ public static String wordWrap(String string, int maxCharsPerLine) {
 		}
 		return document.toString();
 	}
+	
+	/**
+	 * @since 2.9
+	 */
+	public static List<String> splitIntoLines(String text) {
```

Comment:
That wasn't available in Guava10, but com.google.common.io.CharStreams.readLines(Readable) along with new StringReader(..) will do the trick